platform-tests/src/acceptance/plain_http_test.go

```diff
@@ -0,0 +1,52 @@
+package acceptance_test
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("plain HTTP requests", func() {
```

Comment:
This should be ```Describe("Http client",...```

Reply:
Why? I think it's clearer like this because it describes the scope of these tests.